src/bootstrap/transactions-sheet-boostrapper.js

```diff
@@ -0,0 +1,61 @@
+class AllTransactionsSheetBootstrapper extends BaseSheetBootstrapper {
+  constructor(categoryValidationRule) {
+    super('All-Transactions');
+    this.categoryValidationRule = categoryValidationRule;
+  }
+
+  createSheet() {
+    const numRows = 2000;
+    this.mainTable = new DataTable(
+      this.sheet,
+      1,
+      TransactionsSheet.columnDate,
+      'All-Transactions',
+      TransactionsSheet.headers,
+      numRows
+    )
+      .withDataValidationRules([
+        new ColumnValidationRule(TransactionsSheet.columnCategory, this.categoryValidationRule),
+      ])
+      .initialize([]);
+
+    if (this.mainTable.getHeadersAndDataRange().getFilter() === null) {
+      this.mainTable.getHeadersAndDataRange().createFilter();
+    }
+
+    const image = this.sheet.insertImage(REFRESH_BUTTON_IMAGE, 15, 20);
+    image.assignScript('refresh');
+  }
+
+  applyFormat() {
+    this.sheet.clearFormats();
+    this.sheet.setHiddenGridlines(true);
+    Formatter.applyDefaultTableFormat(this.mainTable);
+
+    const protections = this.sheet.getProtections(SpreadsheetApp.ProtectionType.RANGE);
+    for (let i = 0; i < protections.length; i += 1) {
+      protections[i].remove();
+    }
+
+    this.sheet.protect().setWarningOnly(true).setUnprotectedRanges([this.mainTable.getDataRange()]);
+
+    this.sheet.setColumnWidth(1, 20);
+    this.sheet.setColumnWidth(TransactionsSheet.columnDescription, 250);
+    this.sheet.setColumnWidth(TransactionsSheet.columnCategory, 150);
+    this.sheet.setColumnWidth(TransactionsSheet.columnKey, 250);
+
+    this.sheet.setFrozenRows(3);
+
+    Formatter.applyDataFormat(this.mainTable, TransactionsSheet.transactionsFormat);
+
+    const conditionalFormatRules = [
+      SpreadsheetApp.newConditionalFormatRule()
+        .whenTextEqualTo('Other')
+        .setFontColor('#F46524')
+        .setBold(true)
+        .setRanges([this.mainTable.getDataRange()])
+        .build(),
+    ];
+    this.sheet.setConditionalFormatRules(conditionalFormatRules);
```

Comment:
For some reason this conditional format rule is not working. In order to avoid migrations, let's get it working first.

Reply:
And given all transactions show up in the same table, we could use different color schemes to easily identify what's income, investment and expenses

Reply:
hum,  I do not see the reason to use different schemes as we have filters now. What would be the purpose of color schemes?

Reply:
Fair enough when using filters. But when going through the transactions list as a whole (i.e. it is not correctly classified), using green for negative values (representing income) and light red for positives (representing expense) would be helpful.

Reply:
this only happens with CSV (negative vs positive) if I use Plaid it will put all numbers as positives automatically that will make the color scheme wrong, any ideas?